### PR TITLE
Add bump.sh

### PR DESCRIPTION
**Pull request by fmerian**
"Bump.sh helps you build a branded single source of truth, documenting and cataloging your APIs. Teams now have a reference point for building and consuming APIs whether with OpenAPI, Swagger, or AsyncAPI."

### CODE REVIEW

**Review by agamm — APPROVED** [review]
Looks great! thanks @fmerian

**Comment by fmerian** [comment]
awesome! ping @scharrier for the record

**Comment by scharrier** [comment]
Thanks a lot for the add @fmerian 🙏

File: README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 
 ## Documentation
 *Documentation solutions.*
+* [Bump.sh](https://bump.sh/) - API change management solution.
```
Comment (scharrier) [inline README.md:148]: Here is a small suggestion that could better match our description:

```suggestion
* [Bump.sh](https://bump.sh/) - API documentation and change management solution.
```

**Comment by agamm** [comment]
Thanks @fmerian and @scharrier

**Pull request merged**